TrackingTools/PatternTools/interface/Trajectory.h

```diff
@@ -357,6 +368,7 @@ class Trajectory
   float theChiSquaredBad=0;
 
   signed short theNumberOfFoundHits=0;
+  signed short theNumberOfFoundPixelHits=0;
```

Comment:
Wouldn't this name be confusing for Phase2 developments,
or do you expect it to cover the OT pixel layer hits as well? .. or will we call them differently?